xtas/tasks/_polarity.py

```diff
@@ -99,7 +99,7 @@ def classify(doc):
             MODEL = load(model_path)
         except (IOError, OSError) as e:
             if e.errno == errno.ENOENT:
-                MODEL = train()
```

Comment:
Right, I'll leave it to you to decide. I think splitting would be best, but that changes the API. Then again so would removing it.